package.json

```diff
@@ -51,7 +51,7 @@
     "useLocales": {
       "type": "boolean",
       "default": true,
-      "description": "If checked, then the locales below will be used to check words using Hunspell.",
+      "description": "If checked, then the locales below will be used for checking.",
```

Comment:
`If checked, then the locales below will be used for checking.` Is still wrong for 
windows correct? 
Isn't it more along the line of `Use the system default dictionary, or the locales listed below for checking`

Reply:
"Use System" doesn't work for at least 90% of the Window installations. It may not work for the other 10%, I'm just not entirely sure since I can't exhaustively test all the versions/environments. My gut feeling is that "Use System" only works for Mac which is why I removed the Hunspell reference in the configuration.

Reply:
... in the configuration for "Use Locales" and "Locales".

Reply:
I've checked two Windows installations now and haven't gotten "Use System" to provide useful checks.

Reply:
>"Use System" doesn't work for at least 90% of the Window installations. It may not work for the other 10%, I'm just not entirely sure since I can't exhaustively test all the versions/environments. My gut feeling is that "Use System" only works for Mac which is why I removed the Hunspell reference in the configuration.

Yep, in my case as well (tested on  3xWin10{pro,home}) `"Use System"` doesn't do any thing.. 

As someone with no knowledge of the internals, its just a small case of puzzlement when I read the `Use Locales` text, and then don't have to enter anything into the list for my default dictionary to be picked up..  But since both options are enabled by default, it should be fine!